OS X uses security/ instead of pam/ as of Snow Leopard
--- infinoted/infinoted-pam.c.orig	2011-07-17 04:17:04.000000000 -0500
+++ infinoted/infinoted-pam.c	2011-11-02 22:09:29.000000000 -0500
@@ -26,11 +26,7 @@
 #ifdef LIBINFINITY_HAVE_PAM
 
 #include <infinoted/infinoted-pam.h>
-#ifdef __APPLE__
-#include <pam/pam_appl.h>
-#else
 #include <security/pam_appl.h>
-#endif
 #include <sys/types.h>
 #include <grp.h>
 #include <pwd.h>
